deployment/terraform/participant/main.tf

```diff
@@ -303,3 +334,81 @@ resource "local_file" "registry_entry" {
   })
   filename = "${path.module}/build/${var.participant_name}.json"
 }
+
+resource "local_file" "sdd" {
```

Comment:
Consider extracting this part into an own terraform file for better readability.